### PR TITLE
Fixing latex

### PR DESCRIPTION
**Pull request by chinmaySinghal**
Changed the capitalized letters in the equations and fixed some grammar.

### CODE REVIEW

File: docs/en/week01/01-1.md [hunk 1]
```diff
@@ -61,21 +61,21 @@ The more efficient way is called Stochastic Gradient Descent (SGD). Since we wan
 
 ### [Computing gradients by backpropagation](https://www.youtube.com/watch?v=0bMe_vCZo30&t=2336s)
 
-Computing gradients by backpropagation is a practical application of the chain rule. The backpropagation for state gradients is as follows:
+Computing gradients by backpropagation is a practical application of the chain rule. The backpropagation equation for the input gradients is as follows:
```
Comment (Atcold) [inline docs/en/week01/01-1.md]: @skyil7, see update to weird terminology.

File: docs/en/week01/01-1.md [hunk 2]
```diff
@@ -61,21 +61,21 @@ The more efficient way is called Stochastic Gradient Descent (SGD). Since we wan
...
 
 $$
 \begin{aligned}
-\frac{\partial C}{\partial X_{i - 1}} &= \frac{\partial C}{\partial X_i}\frac{\partial X_i}{\partial X_{i - 1}} \\
-\frac{\partial C}{\partial X_{i - 1}} &= \frac{\partial C}{\partial X_i}\frac{\partial F_i(X_{i - 1}, W_i)}{\partial X_{i - 1}}
+\frac{\partial C}{\partial \textbf{x}_{i - 1}} &= \frac{\partial C}{\partial \textbf{x}_i}\frac{\partial \textbf{x}_i}{\partial \textbf{x}_{i - 1}} \\
+\frac{\partial C}{\partial \textbf{x}_{i - 1}} &= \frac{\partial C}{\partial \textbf{x}_i}\frac{\partial f_i(\textbf{x}_{i - 1}, \textbf{w}_i)}{\partial \textbf{x}_{i - 1}}
```
Comment (Atcold) [inline docs/en/week01/01-1.md]: Use `\vect{}` (if it works) or `\boldsymbol{}` if it doesn't.

File: docs/en/week01/01-1.md [hunk 3]
```diff
@@ -61,21 +61,21 @@ The more efficient way is called Stochastic Gradient Descent (SGD). Since we wan
...
 
-The backpropagation for weight gradients is as follows:
+The backpropagation equation for the weight gradients is as follows:
 
 $$
 \begin{aligned}
-\frac{\partial C}{\partial W_{i}} &= \frac{\partial C}{\partial X_i}\frac{\partial X_i}{\partial W_{i}} \\
-\frac{\partial C}{\partial W_{i}} &= \frac{\partial C}{\partial X_i}\frac{\partial F_i(X_{i - 1}, W_i)}{\partial W_{i}}
+\frac{\partial C}{\partial \textbf{w}_{i}} &= \frac{\partial C}{\partial \textbf{x}_i}\frac{\partial \textbf{x}_i}{\partial \textbf{w}_{i}} \\
+\frac{\partial C}{\partial \textbf{w}_{i}} &= \frac{\partial C}{\partial \textbf{x}_i}\frac{\partial f_i(\textbf{x}_{i - 1}, \textbf{w}_i)}{\partial \textbf{w}_{i}}
```
Comment (Atcold) [inline docs/en/week01/01-1.md]: Same, here. `\vect{}` or `\boldsymbol{}` if it does not work.
You may want to try running the website locally to test it out.

Comment (chinmaySinghal) [reply docs/en/week01/01-1.md]: Done. `\vect{}` doesn't work and so I've used `\boldsymbol{}`

File: docs/en/week01/01-1.md (hunk 1 above)
Comment (skyil7) [reply docs/en/week01/01-1.md]: Just perfect.
I will update it in Korean.

File: docs/en/week01/01-1.md (hunk 3 above)
Comment (Atcold) [reply docs/en/week01/01-1.md]: Can you plz apply this correction to all translations?

Comment (chinmaySinghal) [reply docs/en/week01/01-1.md]: Sure!

**Pull request merged**